fleetspeak/test-package.sh

```diff
@@ -23,3 +23,14 @@ apt install $1
 
 /bin/echo 'ls -l /etc/fleetspeak-server'
 ls -l /etc/fleetspeak-server
+
+# At this point the service is doen, since right after the installation it was
+# started without a configuration
+/bin/echo 'systemctl restart fleetspeak-server'
```

Comment:
nit

'set +x' is the cleaner, standard way of accomplishing this.

Reply:
Using set -ex at the top of the file now.